### PR TITLE
feat: Add network change injection method

### PR DESCRIPTION
**Pull request by jinoosss**

### CODE REVIEW

File: packages/adena-extension/src/components/approve/approve-adding-network/approve-adding-network-table/approve-adding-network-table.spec.tsx [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+import theme from '@styles/theme';
+import { GlobalStyle } from '@styles/global-style';
+import ApproveAddingNetworkTable, { ApproveAddingNetworkTableProps } from './approve-adding-network-table';
+
+describe('ApproveAddingNetworkTable Component', () => {
+  it('ApproveAddingNetworkTable render', () => {
+    const args: ApproveAddingNetworkTableProps = {
+      name: 'Teritori Testnet',
+      rpcUrl: 'https://testnet.gno.teritori.com:26658/',
+      chainId: 'teritori-3',
```
Comment (r3v4s) [inline packages/adena-extension/src/components/approve/approve-adding-network/approve-adding-network-table/approve-adding-network-table.spec.tsx:14]: teritori rpc `https://testnet.gno.teritori.com:26658/status?` returns `teritori-1`

File: packages/adena-extension/src/components/approve/approve-adding-network/approve-adding-network-table/approve-adding-network-table.stories.tsx [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+export default {
+  title: 'components/approve-adding-network/ApproveAddingNetworkTable',
+  component: ApproveAddingNetworkTable,
+} as Meta<typeof ApproveAddingNetworkTable>;
+
+export const Default: StoryObj<ApproveAddingNetworkTableProps> = {
+  args: {
+    name: 'Teritori Testnet',
+    rpcUrl: 'https://testnet.gno.teritori.com:26658/',
+    chainId: 'teritori-3',
```
Comment (r3v4s) [inline packages/adena-extension/src/components/approve/approve-adding-network/approve-adding-network-table/approve-adding-network-table.stories.tsx:13]: same here

**Review by r3v4s — APPROVED** [review]

File: packages/adena-extension/src/components/approve/approve-adding-network/approve-adding-network-table/approve-adding-network-table.spec.tsx (hunk 1 above)
Comment (jinoosss) [reply packages/adena-extension/src/components/approve/approve-adding-network/approve-adding-network-table/approve-adding-network-table.spec.tsx:14]: changed to empty.

File: packages/adena-extension/src/components/approve/approve-adding-network/approve-adding-network-table/approve-adding-network-table.stories.tsx (hunk 2 above)
Comment (jinoosss) [reply packages/adena-extension/src/components/approve/approve-adding-network/approve-adding-network-table/approve-adding-network-table.stories.tsx:13]: changed to empty.

**Review by r3v4s — APPROVED** [review]

**Pull request merged**